src/lib/util.ts

```diff
@@ -231,6 +231,18 @@ export function parseToNumberString(num: number, getString: GetStringFunction):
 	return `${num}`
 }
 
+export const locales = readdirSync("./strings")
+
+export function discordLocaleToBotLocale(discordLocale: string): string {
+	if (discordLocale === "uk") discordLocale = "ua"
+	const localeWithCountryCode = discordLocale.replace(/([a-z]+)(?:-([A-Z]+))?/, (_, a, b) => (b ? a + b.toLowerCase() : a)),
+		locale = discordLocale.replace(/([a-z]+)?/, (_, a) => a)
```

Comment:
```suggestion
		locale = discordLocale.replace(/([a-z]+)(?:-([A-Z]+))?/, (_, a) => a)
```
This won't work

Reply:
You're right lol. Pretty strange, it looks like it would work. Well it's fixed now.